/ColorDetector:
  ros__parameters:
    class_name: blue
    upper_hsv_list: [0, 100, 100]
    lower_hsv_list: [10, 255, 255]
    min_area: 75
